android/res/layout/place_page_button_frame.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="match_parent"
+  android:layout_height="@dimen/place_page_buttons_height"
+  android:layout_marginTop="@dimen/margin_base"
+  android:orientation="horizontal"
+  android:background="?ppButtonsBackground"
+  android:clickable="true"
```

Comment:
Ок
> On May 27, 2016, at 4:51 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/layout/place_page_button_frame.xml <https://github.com/mapsme/omim/pull/3347#discussion_r64907169>:
> 
> > @@ -0,0 +1,11 @@
> > +<?xml version="1.0" encoding="utf-8"?>
> > +<LinearLayout
> > +  xmlns:android="http://schemas.android.com/apk/res/android"
> > +  android:layout_width="match_parent"
> > +  android:layout_height="@dimen/place_page_buttons_height"
> > +  android:layout_marginTop="@dimen/margin_base"
> > +  android:orientation="horizontal"
> > +  android:background="?ppButtonsBackground"
> > +  android:clickable="true"
> Там были паддинги по краям, сквозь которые просачивались нажатия. По новому дизайну их убрал, clickable, соответственно, тоже. Будет в следующем коммите.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/mapsme/omim/pull/3347/files/ceb41180a07558972e9005df42fbf58f3591e034#r64907169>, or mute the thread <https://github.com/notifications/unsubscribe/ACKTROsifvaI3haY8Xih--cBL2nq3dKMks5qFvb2gaJpZM4InaE2>.
>